scipy/stats/tests/test_qmc.py

```diff
@@ -534,23 +535,40 @@ def test_sample(self, *args):
         pytest.skip("Not applicable: the value of reference sample is"
                     " implementation dependent.")
 
+    @pytest.mark.parametrize("orthogonal", [False, True])
+    @pytest.mark.parametrize("centered", [False, True])
     @pytest.mark.parametrize("optimization", [None, "random-CD"])
-    def test_sample_stratified(self, optimization):
-        d, n = 4, 20
+    def test_sample_stratified(self, optimization, centered,
+                               orthogonal):
+        d, n = 4, 25
         expected1d = (np.arange(n) + 0.5) / n
         expected = np.broadcast_to(expected1d, (d, n)).T
 
-        engine = self.engine(d=d, scramble=False, centered=True,
+        engine = self.engine(d=d, scramble=False, centered=centered,
+                             orthogonal=orthogonal,
                              optimization=optimization)
```

Comment:
I think that tests for specific subclasses of `QMCEngine` should be written in a way that is specific to the subclass. For instance, since this code is only for `LatinHypercube`, it seems unusual for it to be referred to as `self.engine` and to explicitly pass in `scramble=False` when `scramble`ing isn't supported by `LatinHypercube`.

Reply:
Agreed I will call `LatinHypercube` explicitly.